apps/website/contents/resume.md

```diff
@@ -343,6 +343,66 @@ Simply copy the content from your resume and paste it into a plain text document
 - Characters are displaying incorrectly in plain text
 - Sections are disorganized
 
+## Crafting the Perfect Cover Letter
```

Comment:
Use "Sentence case" for all the headings please

Reply:
✅ Fixed. Updated all titles to be consistent with "Sentence Case".